build/media_source/com_media/scss/components/_media-browser.scss

```
@@ -311,7 +311,7 @@
     position: relative;
     z-index: 0;
     width: 50px;
-    padding: 0;
+    padding: 0.6rem 0;
```

```suggestion
    padding: .6rem 0;
```
Please fix this so the scss style linter is happy and not reports "Unexpected leading zero" anymore.

---

I allow myself to commit my suggestion.